modules/apps/object/object-test/src/testFunctional/macros/ObjectAdmin.macro

```diff
@@ -725,6 +725,10 @@ definition {
 			locator1 = "ObjectAdmin#FIELDS_RADIO_OPTION");
 	}
 
+	macro clickModalFormulaBuilder {
+		Click(locator1 = "ObjectAdmin#MODAL_FORMULA_BUILDER_BUTTON");
+	}
+
```

Comment:
We don't need to create a macro for this since it's only one function. You can just call the function directly in the test case